standalone/src/models/cart/cart/discounted-line-item-price-for-quantity/fields-config.ts

```diff
@@ -9,7 +9,7 @@ import type {
 } from './types';
 
 const commonFieldsConfig = {
-  quantity: fake((f) => f.number.int()),
+  quantity: fake((f) => f.number.int({ min: 1, max: 10 })),
```

Comment:
The number here represents the amount of items in the cart with discounts and we were using very high numbers for it.